### DISCUSSION TITLE
getting messages on mqtt, but devices (and tasmota) not showing up in home assistant

### DISCUSSION THREAD

**Discussion by rmalchow** [arendst/Tasmota#11482] [Q&A] [answered]
hi,

i have a sonoff bridge flashed with tasmota 9.3.1.2, and i have already sucessfully paired a device (osram smart switch) and configured mqtt.

when i subscribe to /tele on the broker, i see button presses like:

```
tele/tasmota_01/SENSOR {"ZbReceived":{"0x5000":{"Device":"0x5000","0006!00":"","Power":0,"Endpoint":2,"LinkQuality":92}}}
```

and i also see this: 
```
tele/tasmota_01/LWT Offline
tele/tasmota_01/LWT Online
```
during reboots. 

i _also_ have zigbee2mqtt running on the same broker - but with the default "zigbee2mqtt"  base topic. 

i have then added the tasmota integration on home assistant - which is already working with zigbee2mqtt - but i am not getting the status sensors for tasmota, and i am also not seeing the device paired to the bridge. 

is it possibly a problem that zigbee2mqtt is using the same broker? what else could possibly be wrong?

**Comment by ascillato2** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you.

Remember to read the [Contributing Guideline and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md). Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
